hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/client/rpc/TestReadRetries.java

```diff
@@ -241,16 +233,6 @@ public void testPutKeyAndGetKeyThreeNodes()
     factory.releaseClient(clientSpi, false);
   }
 
-  private void verifyIntermediateDir(OzoneBucket bucket,
```

Comment:
removed this because this expects an Exception while performing getKey on a directory which has been changed as part of this PR.

Reply:
Thanks @sadanand48 for the clear explanation. This test were running with below params. This gives me the impression that with existing code `enableFileSystemFlag=true`, it expects exception should be thrown for a dir.

enableFileSystemFlag=true and layoutVersion=V0
enableFileSystemFlag=true and layoutVersion=V1

I think, we need to get the correct functionality or the expectation with enableFileSystemFlag=true or false.

Reply:
Resolving this based on offline discussion